examples/ExampleApp/app/src/main/kotlin/io/embrace/android/exampleapp/ui/examples/UserExample.kt

```diff
@@ -0,0 +1,72 @@
+package io.embrace.android.exampleapp.ui.examples
+
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import io.embrace.android.embracesdk.Embrace
+
+@Composable
+fun UserExample() {
```

Comment:
I guess we'll still need this as long as it hasn't been officially deprecated?

Reply:
Yep - should be trivial enough to remove, and I've avoided less used features such as personas.